File: app/src/main/java/com/song/deviceinfo/ui/store/StoreViewModel.java
package com.song.deviceinfo.ui.store;

import android.content.Context;

import com.song.deviceinfo.info.StoreInfo;
import com.song.deviceinfo.model.beans.StorageBean;
import com.song.deviceinfo.ui.base.BaseViewModel;

import java.util.List;

/**
 * Created by chensongsong on 2020/6/1.
 */
public class StoreViewModel extends BaseViewModel {

    protected List<StorageBean> getStoreInfo(Context context) {
        return StoreInfo.getStoreInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/store/StoreViewModel.kt
package com.song.deviceinfo.ui.store

import android.content.Context
import com.song.deviceinfo.info.StoreInfo
import com.song.deviceinfo.model.beans.StorageBean
import com.song.deviceinfo.ui.base.BaseViewModel

/**
 * Created by chensongsong on 2020/6/1.
 */
class StoreViewModel : BaseViewModel<StorageBean>() {
    fun getStoreInfo(context: Context): List<StorageBean> {
        return StoreInfo.getStoreInfo(context)
    }
}
